src/proc/occlusion-filter.cpp

```diff
@@ -36,6 +36,29 @@ namespace librealsense
             break;
         }
     }
+   int gcd(int a, int b) {
+       if (b == 0)
+           return a;
+       return gcd(b, a % b);
+   }
+   // Return the gretest common divisor of a 
+   // and b which lie in the given range. 
+   int maxDivisorRange(int a, int b, int l, int h)
+   {
+       int g = gcd(a, b);
+       int res = g;
+
+       // Loop from 1 to sqrt(GCD(a, b). 
+       for (int i = l; i * i <= g && i <= h; i++)
```

Comment:
`i <= h` only is problematic as  `h <l` is not guaranteed. Use min (l,h)
imo the lookup should go between 1..(GCD(a, b).